docs/crds/gslbhostrule.md

```diff
@@ -58,11 +64,13 @@ If this field is not provided in `GSLBHostRule`, the site persistence property w
 
 7. `trafficSplit`: Specify traffic steering to member clusters/sites. The traffic is then split proportionately between two different clusters. Weight for each cluster must be provided between 1 to 20. If not added, GDP object's traffic split applies on this GslbService.`trafficSplit` can also be used to prioritize certain clusters before others. Maximum value for priority is 100 and default is 10. Let's say two clusters are given a priority of 20 and a third cluster is added with a priority of 10. The third cluster won't be routed any traffic unless both cluster1 and cluster2 (with priority 20) are down.
 
-8. `ttl`: Override the default `ttl` value specified on the GDP object using this field.
+8. `publicIP`: An optional public IP address can be specified for each site. This field is used to host the public IP address for the VIP, which gets NAT’ed to the private IP by a firewall. Please check [here](https://avinetworks.com/docs/21.1/nat-aware-public-private-configuration) for more details.
```

Comment:
It will be better to change `21.1` to `latest` so this document will always point to the latest document.

Reply:
Changed the link